test/spec/modules/lemmaBidAdapter_spec.js

```diff
@@ -192,6 +193,7 @@ describe('lemmaBidAdapter', function() {
         expect(data.site.publisher.id).to.equal(bidRequests[0].params.pubId.toString()); // publisher Id
         expect(data.imp[0].tagid).to.equal(undefined); // tagid
         expect(data.imp[0].bidfloorcur).to.equal(bidRequests[0].params.currency);
+        expect(data.imp[0].bidfloor).to.equal(bidRequests[0].params.floor);
```

Comment:
Here you are also comparing `undefined` with `undefined`.

Reply:
Added test coverage for getFloor function changes.